editor/editor_properties.cpp

```diff
@@ -258,7 +258,7 @@ EditorPropertyPath::EditorPropertyPath() {
 
 ///////////////////// MEMBER /////////////////////////
 
-void EditorPropertyMember::_property_selected(const String &p_selected) {
+void EditorPropertyMember::_property_selected(const String &p_selected, const String &p_category /*= ""*/) {
```

Comment:
I wouldn't put the default argument as a comment inline like that, it looks weird. But actually `p_category` isn't used anywhere, so it does not need to be added.

Reply:
Is removed.